pkg/inngest/function.go

```diff
@@ -75,6 +77,11 @@ type Function struct {
 	Edges []Edge `json:"edges,omitempty"`
 }
 
+type Debounce struct {
+	Key    *string `json:"key"`
```

Comment:
No key will just mean debounce will apply to all function runs for this function, correct?

Reply:
Yep